client/lib/credit-card-details/validation.js

```diff
@@ -89,19 +103,20 @@ function validateCreditCard( cardDetails ) {
 	} );
 }
 
-function creditCardValidator( /* validationProperties... */ ) {
-	var validationProperties = toArray( arguments );
-
+function creditCardValidator( ...validationProperties ) {
 	return {
 		isValid: function( value, cardDetails ) {
-			var validationResult;
 			if ( ! value ) {
 				return false;
 			}
 
-			validationResult = validateCreditCard( cardDetails );
+			const validationResult = validateCreditCard( cardDetails );
 
 			return validationProperties.every( function( property ) {
+				if ( property === 'notExpired' ) {
```

Comment:
I haven't come up with a better idea here. For some reasons the external library we use here validates expiration date the other way around comparing to all other checks ;)